src/renderer/helpers/fp/eq.test.ts

```diff
@@ -28,23 +28,23 @@ describe('helpers/fp/eq', () => {
 
   describe('eqAsset', () => {
     it('is equal', () => {
-      const a = ASSETS_TESTNET.RUNE
+      const a = ASSETS_TESTNET.BNB
       expect(eqAsset.equals(a, a)).toBeTruthy()
     })
     it('is not equal', () => {
-      const a = ASSETS_TESTNET.RUNE
```

Comment:
`ASSETS_TESTNET.RUNE` -> `AssetNativeRune`
(here and for other changes in this file)